packages/snaps-sdk/src/jsx/validation.ts

```diff
@@ -86,7 +88,7 @@ import {
   svg,
   typedUnion,
 } from '../internals';
-import type { EmptyObject } from '../types';
+import type { CaipChainId, EmptyObject } from '../types';
```

Comment:
This should be imported from `@metamask/utils`. The re-export from `@metamask/snaps-sdk` is mainly for convenience for Snap developers.